packages/sampler/src/MoonstoneEnvironment/MoonstoneEnvironment.js

```diff
@@ -157,11 +174,15 @@ const StorybookDecorator = (story, config) => {
 			className={classnames(classes)}
 			title={`${config.kind} ${config.story}`.trim()}
 			description={config.description}
-			locale={selectV2('locale', locales, 'en-US', globalGroup)}
-			textSize={boolean('large text', Config, (getPropFromURL('large text') === 'true')) ? 'large' : 'normal'}
-			highContrast={boolean('high contrast', Config, (getPropFromURL('high contrast') === 'true'))}
-			style={backgroundLabelMap[select('background', backgroundLabels, Config, getPropFromURL('background'))]}
-			skin={select('skin', skins, Config, getPropFromURL('skin'))}
+			locale={select('locale', locales, Config)}
+			textSize={boolean('large text', Config, getKnobFromArgs(args, 'large text')) ? 'large' : 'normal'}
+			highContrast={boolean('high contrast', Config, getKnobFromArgs(args, 'high contrast'))}
+			style={{
+				'--moon-env-background': backgroundLabelMap[select('background', backgroundLabels, Config, getKnobFromArgs(args, 'background'))]
+				// '--moon-env-background-filter': 'blur(' + number('background blur', Config, {range: true, min: 0, max: 10, step: 0.5}, 3) + 'px)'
```

Comment:
Why have we left this (and L184) in here commented out? Should we remove or add a descriptive comment as to why they exist?

Reply:
I wanted an opinion on whether a dynamic blur knob would be useful. I left this in place in case we wanted to add it at some point (it would already be there), since the blur feature is new, this would allow configurability to it. Wanted people to use it and think about the effect before answering about the blur amount.

Reply:
Good idea to have a conversation about adding the feature. I think it would be helpful in the future to explicitly call that out in the PR description and possibly include it by default. As commented code, it appears to be cruft and doesn't provide the context around which to have a conversation.

What features of the framework will be better demonstrated by adding this knob?

Reply:
Well, I didn't necessarily want to have the conversation right now, since nobody has seen or used storybook with the blurred background yet, so it's really just a passive reminder for a future conversation.

If it _did_ actively demonstrate a real feature of the framework, it would have been included (not commented out), but it doesn't directly demonstrate that. However, the ability to customize the blur amount may be helpful to someone wanting to see just how "crisp" a background could be before a component looks like it's hard to read. It's also sort of _neat_ (read: not mandatory) to see how a background blurring effect can affect foreground readability/usability and be adjusted dynamically.

Reply:
I'm concerned that including commented code as a means to prompt future conversation is not particularly effective. I would recommend either including the feature in the PR for discussion or opening a follow up ticket to discuss and come to a decision.

Reply:
Yeah, this wasn't meant to be a conversation topic, just some parked code for a future feature that I didn't have to store, archive, regurgitate or rewrite if the need came up. I'll just delete it. It's already got far more attention than I had intended or even any of the rest of the code or changes.